subsys/fs/nvs/nvs.c

```diff
@@ -73,6 +77,35 @@ static int nvs_flash_al_wrt(struct nvs_fs *fs, uint32_t addr, const void *data,
 	return rc;
 }
 
+static int nvs_flash_al_wrt(struct nvs_fs *fs, uint32_t addr, const void *data,
+			     size_t len)
+{
+	int rc;
+
+#ifdef CONFIG_NVS_WRITE_RETRY
+	for (int i = 0; i < 2; i++) {
+#endif
```

Comment:
Maybe do `#else` with just opening bracket. This way the ifdef in 103-105 could be avoided, although I do not know whether style allows that.